File: template/common/src/client/java/com/example/modid/client/ClientInit.java
package com.example.modid.client;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

// physical-client logic starts here
public final class ClientInit {
    private static final Logger LOGGER = LoggerFactory.getLogger("modid/client");

    private ClientInit() {}

    public static void init() {
        LOGGER.info("modid client init");
    }
}

File: template/common/src/main/kotlin/com/example/modid/ClientInit.kt
package com.example.modid.client

import org.slf4j.LoggerFactory

// physical-client logic starts here
object ClientInit {
  private val logger = LoggerFactory.getLogger("modid/client")

  @JvmStatic
  fun init() {
    logger.info("modid client init")
  }
}
